### ISSUE TITLE
Fix The Netherlands on the map

### ISSUE THREAD

**Issue by Tralapo** [piwik/piwik#9170]
The visitor map in Piwik isn't very accurate and I'm fine with that, but you guys are quite far off with The Netherlands. It's true we do claim land from water sometimes, but in Piwik it's quite extreme. The 'IJsselmeer', a big lake in the middle of the country, has been turned into land in the Piwik Map. 

Not a big priority I would say, but would be good to look at.

Piwik:
![Piwik](https://cloud.githubusercontent.com/assets/11296429/10967304/d885c67a-83b9-11e5-9448-222e19256e36.JPG)

Actual Netherlands:
![kaart nederland met provincies](https://cloud.githubusercontent.com/assets/11296429/10967340/1c6a1ad0-83ba-11e5-8c90-bc5406dfbce0.jpg)

**Comment by tsteur** [comment]
I tried to find out if this is fixed with latest Kartograph version ( https://github.com/kartograph/kartograph.js ) but couldn't find a demo. Does anyone know where we get updated SVG files? It seems like they are not in the kartograph project itself

**Comment by sgiehl** [comment]
Afaik they were built using https://github.com/gka/piwik-map-generator
Tried to fork and rebuild them once, but didn't get it running proper...

**Comment by tsteur** [comment]
Yep seems somehow broken, already fails when trying to download some ZIP files

**Comment by sgiehl** [comment]
Tried to fixed that... See https://github.com/sgiehl/piwik-map-generator

**Comment by tsteur** [comment]
Wow, cool. Let me know if you get it working or need any help. Maybe we can also contact @gka if we don't get it to work

**Comment by sgiehl** [comment]
Tried that in February... Didn't get it working proper and @gka didn't respond. See #321

**Comment by gka** [comment]
the map is not "wrong", it's just a matter wether or not the interior lakes are being cut-out.

As for the Piwik map I think we used NaturalEarth adm_level1_10m shapefiles as input data, in which the Netherlands happen to look like this.

![image](https://cloud.githubusercontent.com/assets/617518/11076600/9e3cb078-87c9-11e5-8fd8-98a28164d5f2.png)

To fix it, just add a custom shapefile for the Netherlands in the `custom` folder and it'll be used instead of the NE shapefile.

**Comment by gka** [comment]
btw, I moved the repo to [piwik/piwik-map-generator](https://github.com/piwik/piwik-map-generator) to make it easier for everyone to maintain it.

**Comment by sgiehl** [comment]
@gka do you have some time to adjust the repo so it works again? Guess the urls needs to be adjusted

**Comment by mattab** [comment]
Hi @gka, thanks for moving the project to the repo!
